animesaturn.py

```diff
@@ -0,0 +1,336 @@
+            new_r = requests.get(url = URL, params = {})
+            pastebin_url = new_r.text
+            parsed_html = BeautifulSoup(pastebin_url,"html.parser")
+            anno = parsed_html.find('div', attrs={'class':'container shadow rounded bg-dark-as-box mb-3 p-3 w-100 text-white'})
+            release = re.findall("(?<=<b>Data di uscita:</b> )(.*)(?=<br/>)",str(anno))
+            self.anime[release[0]] = URL
+        locale.setlocale(locale.LC_TIME, 'it_IT.UTF-8')
+        ordered_data = sorted(self.anime.items(), key = lambda x:datetime.strptime(x[0], "%d %B %Y"), reverse=False)
+        self.titolo = re.findall("(?<=anime/)(.*)", ordered_data[0][1])[0]
+        for x in ordered_data:
+            self.only_link.append(x[1])
+        with concurrent.futures.ThreadPoolExecutor(max_workers=len(self.only_link)) as pool:
+            if(self.debug):
+                for t in pool._threads:
+                    print(t)
+            pool.map(self.selected_anime, self.only_link)
+
+
+    def search(self,name):
+        anime_list  = list()
+        URL = "https://www.animesaturn.it/animelist"
+        r = requests.get(url = URL, params = {"search":name})
+        pastebin_url = r.text
+        html = pastebin_url
+        parsed_html = BeautifulSoup(html,"html.parser")
+        animes = parsed_html.find_all('ul', attrs={'class':'list-group'})
+        x = 1
+        if (len(animes)) == 0:
+            print("\x1b[31mNessun Anime trovato per %s\x1b[0m"%name)
+            exit(0)
+        for dim in animes:
+            print("--------")
+            print(x)
+            print("TITOLO:")
+            print("\x1b[32m" + dim.find('a',attrs={'class':'badge badge-archivio badge-light'}).text + "\x1b[0m")
+            print("TRAMA:")
+            print("\x1b[37m" + dim.find('p',attrs={'class':'trama-anime-archivio text-white rounded'}).text +"\x1b[0m")
+            anime_list.append(dim.find('a')['href'])
+            print("--------")
+            x+=1
+        if(self.test_ID):
+            selected = 1
+        else:
+            while True: #richiedere id se + sbagliato
+                try:
+                    selected = int(input("ID ('0' per uscire):"))
+                    if (selected == 0):
+                        exit(0)
+                    if (selected >  len(animes) or selected < 0):
+                        print("\x1b[31mCi sono solo %d risultati\x1b[0m"%len(animes))
+                        continue
+                    break
+                except ValueError:
+                    print("\x1b[31mNon è un ID valido, riprovare...\x1b[0m")
+        #selected = 2
+        selected -=1 #la lista parte da 0
+        URL = anime_list[selected]
+        self.season = 1
+        self.selected_anime(URL)
+        if  (self.file_type == 0):
+            self.create_crawl()
+        elif  (self.file_type == 1):
+            self.downloader()
+        else:
+            sys.exit(0)
+    def check_Path(self,crawl_path):
```

Comment:
```suggestion
    @staticmethod
    def check_Path(crawl_path):
```
Codacy: method could be a function